README.md

```diff
@@ -895,6 +895,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [The Odin Project](https://www.theodinproject.com/) - Free, open-source platform with a curriculum focused on JavaScript and Ruby for web development.
   * [Free ATS Resume AI Solutions](https://ayehigh.com) - Free platform with lots of GenAI Powered Resume Solutions helpful for Analyzing, Rewriting, Grooming and Shortlisting Resumes and help in Career Growth.
   * [Free Professional Resume Templates & Editor](https://www.overleaf.com/latex/templates/tagged/cv) - Free platform with lots of Resume templates of Experienced Professionals, ready to clone and edit fully and download, ATS optimized.
+  * [IndiesReadIt](https://indiesread.it) - Free book directory for indiehackers with a dev/code category.
```

Comment:
Thanks, I dont think this is appropriate for the list sorry.  So will close this one, appreciate the contribution though.